CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+- Support for the Django 5.0
+- Support for the Wagtail 6.0
+- Dropped support for Wagtail < 5.2
```

Comment:
note: these should follow the format used in other release notes.

Leave them as is, as I will reword them anyway

Reply:
Thanks